packages/seccomp-tools/PKGBUILD

```diff
@@ -0,0 +1,85 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=seccomp-tools
+pkgver=v1.6.1.r36.g728ec2f
+ver="${pkgver%%.r*}" 
```

Comment:
this variable is used nowhere

Reply:
The `ver` variable is used on `48`, `53`, `54` rows.